src/api.js

```diff
@@ -0,0 +1,23 @@
+// const API_END_POINT = ""; // todo: Actual end API end point
+const API_END_POINT = "http://0.0.0.0:8080/"; // For running locally, please see note below:
+/*
+	You will have to disable the same origin policy on Chrome for the API to work with the UI.
+	Note - Close all Chrome windows before running this:
+	$ open -a Google\ Chrome --args --disable-web-security --user-data-dir
+	And to re-enable, close all Chrome windows and restart. It should go back to normal
+*/
+
+function makeRequest(url) {
```

Comment:
If the server isn't running, Ithe code returns a nasty error. The code should handle that scenario

Reply:
![screen shot 2017-12-11 at 8 31 24 pm](https://user-images.githubusercontent.com/9705465/33852416-5ea2b4fc-deb2-11e7-8c41-c4a404bcc6bb.png)

Reply:
Sweeet!